src/test/ensemble_tests.cpp

```diff
@@ -2195,3 +2195,146 @@ TEST_F(EnsembleFlowTest, WaitForLoadingPipelineDefinitionFromBeginStatus) {
     t.join();
     t2.join();
 }
+
+const char* configJsonWithNoPipeline = R"(
+{
+    "model_config_list": [
+        {
+            "config": {
+                "name": "dummy",
+                "base_path": "/ovms/src/test/dummy",
+                "target_device": "CPU",
+                "model_version_policy": {"all": {}},
+                "nireq": 1
+            }
+        }
+    ]
+})";
+
+TEST_F(EnsembleFlowTest, RetirePipelineAfterLoading) {
+    std::string fileToReload = "/tmp/ovms_config_file1.json";
```

Comment:
Multiple tests use the same path for temporary file. Are you sure we can run tests in parallel this way?